Probe/receive

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Receive messages from Android app
+
+android_path=./sdcard/Android/data/com.google.firebase.messaging.testing.fcmexternalprobertarget/files/
+
+file_name=$1
```

Comment:
Document any parameters for this script.

Reply:
Also document what this script does in the header.

Reply:
addressed in 765a7ee